### ISSUE TITLE
Running latest AAT policies is producing a global CSS bug

### ISSUE THREAD

**Issue by elizabethsjudd** [IBM/carbon-elements#405]
When running the latest IBMa AAT policies (`IBM_Accessibility_2019_02` and `IBM_Accessibility_WCAG21_2019_02`)

Violation:
```
"wcag21.tech.g1200.styleViewport": "Viewport units used to size text attributes do not rescale."
```

## Environment

> Operating system

OS X

> Browser

All

> Automated testing tool and ruleset

AAT

> Assistive technology used to verify

IBMa Karma Plugin

## Detailed description

> What version of the Carbon Design System are you using?

9.68.15

> What did you expect to happen? 

I should not be getting global violations when running AAT

> What happened instead?

Getting violation that text sizes are defined using viewport units

> What WCAG 2.1 checkpoint does the issue violate?

g1200

## Steps to reproduce the issue

Run `@ibma/karma-ibma` or internal DAP extension  using the above policies on a page with carbon css loaded.

**Comment by asudoh** [comment]
Thank you for reporting @elizabethsjudd! Moved to `carbon-elements` as usage of `vw`/`vh` is seen in component-wide usage in this repo, whereas it's only tooltip and deprecated ILN in `carbon-components`. Sounds that we weed some discussions wrt what the original intent of those and see how they affect a11y.

**Comment by elizabethsjudd** [comment]
@asudoh I've been chatting with Josh in slack and after some more digging I found it in the duo type package which we still haven't been able to swap out. Hopefully once we do that, it will resolve our violations.  It is interesting with it's affects to a11y as I've never heard this before... learn something new everyday!

**Comment by joshblack** [comment]
Same! Thanks again for flagging this, really appreciate it @elizabethsjudd 🙏

**Comment by joshblack** [comment]
Closing this for now, feel free to comment and I can re-open if this persists!
